pagopa/io-app
# chore: [IOPID-2409] Add new animated pictogram on Ingress screen

## Short description
Add new animated pictogram on Ingress screen

## List of changes proposed in this pull request
- Import json of new animated pictogram
- Implemented the component that renders the animated pictogram

## Demo
### iOS
| simulator | physical device A11Y | 
| - | - |
| <video src="https://github.com/user-attachments/assets/bdd6d912-3d09-4bc7-bb98-e9f0ad754c5b"/> | <video src="https://github.com/user-attachments/assets/0ebea284-8a7a-4d87-b2fe-a7893ce70402"/> |

### Android
| physical device classic flow A11Y | physical device no BE connection A11Y | 
| - | - |
|  <video src="https://github.com/user-attachments/assets/7342a609-4261-4d08-b721-fb267060baca"/> | <video src="https://github.com/user-attachments/assets/ffeace94-7a4c-4690-b5b8-6b992565f830"/> |


## How to test
Run the application either in a local or production environment. If you use the local environment, avoid running the dev server so that you can view the full display logic of the ingress screen. If you use the production environment, you'll only see it for a few seconds unless you use Proxyman to block the CDN or backend requests.
